internal/charmstore/store.go

```diff
@@ -252,6 +258,7 @@ func (s *Store) insertEntity(entity *mongodoc.Entity) (err error) {
 			Read:  readPerm,
 			Write: writePerm,
 		},
+		Promulgated: false,
```

Comment:
I suspect it will be rare for us to be adding a base entity that is already promulgated, but I've put in the change anyway.